obs-packaging/qemu-vanilla/qemu-vanilla.spec-template

```diff
@@ -113,7 +113,11 @@ done
 %defattr(-,root,root,-)
 /usr/bin/qemu-vanilla-ga
 /usr/bin/qemu-vanilla-pr-helper
+%ifarch x86_64
```

Comment:
IANARE (I am not an RPM expert :), but is it possible to do something like the following to avoid the conditional logic?

```
/usr/bin/qemu-vanilla-system-%{build_arch}
```

/cc @jcvenegas, @marcov

Reply:
%{build_arch} should return ppc64 NOT ppc64le 
OR
The time when the name is set should be "qemu-vanilla-system-ppc64le"

Reply:
I think from the rpm is the only away we can handle this, I dont have experience maintain multi arch specs.

Reply:
@nitkon have you tried with `%{_arch}` ?

Reply:
@marcov : Tried it here. https://build.opensuse.org/package/live_build_log/home:katacontainers:alpha:ppc64le:1.3.0/qemu-vanilla/CentOS_7/ppc64le

Searches for: qemu-vanilla-system-ppc64le
Actual name: qemu-vanilla-system-ppc64

Reply:
OK, for now we can keep this solution.

Maybe that could be handled with a qemu compile flag.

Reply:
Thought: If you can't do it entirely inside the spec file, you could handle it in `update.sh` and add a new variable to https://github.com/kata-containers/packaging/blob/master/obs-packaging/qemu-vanilla/update.sh#L39.